### ISSUE TITLE
Nil pointer de-reference using version v1.3.1

### ISSUE THREAD

**Issue by HeavyWombat** [homeport/pina-golada#59]
**Describe the bug**
Nil pointer de-reference when trying to access packaged assets in version `v1.3.1`.

**To Reproduce**
Check [`gonut` branch `bump/pina-golada-version`](https://github.com/homeport/gonut/tree/bump/pina-golada-version) to reproduce the problem.

**Expected behavior**
Same as the older version, package and extract work without issues.

**Screenshots**
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x70 pc=0x114923e]

goroutine 1 [running]:
github.com/homeport/pina-golada/pkg/files.WriteToDisk(0x0, 0x0, 0xc0001920c0, 0x3f, 0xc0001a0001, 0xc00019c080, 0x1259df6)
	/Users/mdiester/go/pkg/mod/github.com/homeport/pina-golada@v1.3.1/pkg/files/files_util.go:127 +0x16e
```

**Comment by LynxPlay101** [comment]
I won't be available to fix this issue till Monday. You are free to take care of it if it is urgent for your project.

**Comment by HeavyWombat** [comment]
No worries, there is no rush. I just created the branch and this issues so that I have written it down and do not have to remember it.

**Comment by LynxPlay101** [comment]
Well, the bump/pina-golada-version branch seems to build and pass all its unit tests. I imagine that issue only occurs when the code is actually being run ?

**Comment by LynxPlay101** [comment]
Fixed by #60
